inventory_blocks/prometheus_external_nfs.ini

```diff
@@ -1,8 +1,3 @@
-
-openshift_prometheus_storage_kind=nfs
-openshift_prometheus_storage_host={nfs_server}
-openshift_prometheus_storage_nfs_directory={nfs_export_path}
-openshift_prometheus_storage_volume_name=prometheus
 openshift_prometheus_storage_labels={{'storage': 'prometheus'}}
 openshift_prometheus_storage_type=pvc
 
```

Comment:
The name of this file should be changed to reflect this PR, 
how about prometheus_external_pvs.ini

Reply:
No, it doesn't - that will break the way the inventory creator works. It looks for `{component}_{storage_type}` in the blocks directory. If it won't have that, it'll assume default applies, and the default does not apply in this case because prometheus requires some inventory lines that are not required by other components.

The _external_nfs part is the `storage_type` from the inventory creator. Also note that the rest of the definitions in this file do use external NFS, because there's no problem with running alertbuffer/alertmanager on NFS.

Reply:
Meaning that this ini file serves several blocks  (prometheus, prometheus_alertmanager & prometheus_alertbuffer). which makes it hard to be able to configure each separately (which is what we are trying to do)
Separating it into 3 blocks will probably be required anyway when we move to HA configuration (soon) where each resides in a different POD so it may be better to separate it now (and have the name right)

Reply:
I guess that's possible, but that also means I'll have to add special-casing in the inventory creatopr to prevent disabling these components when prometheus is enabled.

I prefer to make that split only when it'll be absolutely necessary, if you don't mind, because the inventory creator is complicated enough as it is.

Reply:
This is indeed a very small item that can be changed later.